### ISSUE TITLE
Incorrect description: Attribute should be placed before documentation comment.

### ISSUE THREAD

**Issue by mikebronner** [slevomat/coding-standard#1591]
<img width="969" alt="Screenshot 2023-06-30 at 9 34 51 AM" src="https://github.com/slevomat/coding-standard/assets/1791050/41f18ccf-f73d-47e9-945b-3851d643ceb0">
<img width="982" alt="Screenshot 2023-06-30 at 9 36 45 AM" src="https://github.com/slevomat/coding-standard/assets/1791050/9ea9c8a2-185a-428f-b4bd-2ae783ac5016">

The implemented rule is:
```xml
	<rule ref="SlevomatCodingStandard.Attributes.RequireAttributeAfterDocComment"/>
```

The verbiage should probably read "Attribute should be placed after documentation comment."

**Comment by kukulich** [comment]
Thank you for your report.

Fixed in https://github.com/slevomat/coding-standard/commit/fec575dab610bc3d45335b7d5760ae460fde2f72
